utils/utils.go

```diff
@@ -46,3 +66,24 @@ func GetStringFromEnvWithDefault(enVarName, defaultValue string) string {
 
 	return envValue
 }
+
+//MustGetStringFromEnv panics if OS Environment Variable is not set
+func MustGetStringFromEnv(enVarName string) string {
+	envValue := os.Getenv(enVarName)
+	if envValue == "" {
+		panic(fmt.Sprintf("%s must not be empty", enVarName))
+	}
+
+	return envValue
+}
+
+// GetBoolFromEnvWithDefault parses an OS Environment Variable as bool
+func GetBoolFromEnvWithDefault(enVarName string, defaultValue bool) bool {
+	envValue := os.Getenv(enVarName)
+	envBool, err := strconv.ParseBool(envValue)
+	if err != nil {
```

Comment:
I would differentiate between no value (`envValue  == ""`) and a value that cannot be parsed as string (`envValue == "flase"`) to make sure typos in the env vars don't accidentally get the default values and stay unnoticed